packages/@coorpacademy-player-services/src/test/fixtures/recommendations.js

```diff
@@ -1,17 +1,35 @@
+// @flow strict
+
 import reduce from 'lodash/fp/reduce';
 import _find from 'lodash/fp/find';
 
+import type {Level, Recommendation} from '../../types';
 import levelsData from './data/levels';
 import recommendations from './data/recommendations';
 
-const levels = reduce((map, object) => map.set(object.ref, object), new Map(), levelsData);
+const levels: Map<string, Level> = reduce(
+  (map, object) => map.set(object.ref, object),
```

Comment:
I'd prefer using objects instead of map

Reply:
why not but not for this PR : )

Reply:
there are maps in quite every fixtures as of today